OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -571,6 +575,15 @@ public bool CanEnterCell(CPos cell, Actor ignoreActor = null, bool checkTransien
 			return Info.CanEnterCell(self.World, self, cell, ignoreActor, checkTransientActors ? CellConditions.All : CellConditions.BlockedByMovers);
 		}
 
+		public bool ContainsTemporaryBlocker(CPos cell)
```

Comment:
This may fit better in `WorldUtils`. Just needs a `Actor ignoreActor = null` parameter like `CanEnterCell()`.